test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ApplicationInsightsEndToEndTests.cs

```diff
@@ -25,6 +28,8 @@ public class ApplicationInsightsEndToEndTests
     {
         private const string _mockApplicationInsightsUrl = "http://localhost:4005/v2/track/";
         private const string _mockQuickPulseUrl = "http://localhost:4005/QuickPulseService.svc/";
+
+        private readonly TestTelemetryChannel _channel = new TestTelemetryChannel();
```

Comment:
Most of the changes here are to get the e2e tests (except the QuickPulse) using a test Channel, rather than an HttpListener. This is more reliable going forward and has already been changed in `dev`.